apps/drained/boot.ts

```diff
@@ -2,7 +2,7 @@
 const { battery: threshold = 5, interval = 10, keepStartup = true }: DrainedSettings
   = require("Storage").readJSON(`drained.setting.json`, true) || {};
 
-drainedInterval = setInterval(() => {
+(global as any).drainedInterval = setInterval(() => {
```

Comment:
This removes cross-file type safety and makes it easier to introduce a bug if `drainedInterval` is changed in one file and not another, let's back it out.

Happy to have the `declare var` here too if you'd like to also indicate that we have a global variable

Reply:
That is a very good argument that I did not think about, agreed.